lib/locales/pl.ts

```diff
@@ -390,7 +390,7 @@ export const locale: Locale = {
     matthias_apostle: 'Św. Macieja, Apostoła',
     maurice_tornay_priest: 'Bł. Maurycego Tornaya, prezbitera i męczennika',
     maximilian_mary_raymund_kolbe_priest: 'Św. Maksymiliana Marii Kolbego, prezbitera i męczennika',
-    mederic_of_autun_and_droctoveus_of_autun_abbots: 'Św. Mederyka i Droctovæi, opatów',
+    mederic_of_autun_and_droctoveus_of_autun_abbots: 'Świętych Mederyka i Droktoweusza, opatów',
```

Comment:
Again: canonisation name typed in full: why?

Reply:
Same as above.